components/eamxx/cime_config/testdefs/testmods_dirs/eamxx/aurora_run/shell_commands

```diff
@@ -1,6 +1,2 @@
-
-$CIMEROOT/../components/eamxx/scripts/atmchange transport_alg=0 -b
```

Comment:
@oksanaguba do you mind that I switched this testmod to SL? I can revert it to Eulerian if you prefer.

Reply:
thanks, that is ok